cypress/integration/html-reader/injectables.ts

```diff
@@ -0,0 +1,29 @@
+import { IFRAME_SELECTOR } from '../../support/constants';
+
+describe('useHtmlReader configuration settings', () => {
+  it('should have no injectables by default', () => {
+    cy.visit('/test/no-injectables');
+    cy.getIframeHead(IFRAME_SELECTOR).find('link').should('not.exist');
+    cy.getIframeHead(IFRAME_SELECTOR).contains('title', 'Cover');
+  });
+
+  it('should render css injectables when provided', () => {
+    cy.visit('/test/with-injectables');
+    cy.getIframeHead(IFRAME_SELECTOR).contains('title', 'Cover');
+    cy.getIframeHead(IFRAME_SELECTOR)
+      .find(
+        `link[href="${
+          Cypress.config().baseUrl
+        }/fonts/opensyslexic/opendyslexic.css"]`
+      )
```

Comment:
So was this the key here @samanthaandrews? Using `.find` instead of `.within` and `.get`?

Reply:
Also I previously had `href^=...` in order to match an href containing the string so I didn't have to provide the baseUrl and everything. Did that not work for you?